ecosystem/indexer-grpc/indexer-grpc-cache-worker/src/lib.rs

```diff
@@ -18,6 +20,12 @@ pub struct IndexerGrpcCacheWorkerConfig {
     pub file_store_config: IndexerGrpcFileStoreConfig,
     pub redis_main_instance_address: RedisUrl,
     pub enable_verbose_logging: Option<bool>,
+    #[serde(default = "default_storage_format")]
+    pub storage_format: StorageFormat,
+}
```

Comment:
I don't think that we should have this as a flag b/c changing this will cause irreparable downstream impact so this seems like something that should be code-level gated not deployment level.